lib/dynamo-restore.js

```diff
@@ -106,6 +106,18 @@ DynamoRestore.prototype._checkTableExists = function(error, data) {
         // Table exists, should we overwrite it??
         if (this.options.overwrite) {
             this.emit('warning', util.format('WARN: table [%s] will be overwritten.', this.options.table));
+            params = {
+                TableName: this.options.table,
+                ProvisionedThroughput: {
+                    ReadCapacityUnits: this.options.readcapacity,
+                    WriteCapacityUnits: this.options.concurrency
+                }
+            };
+            dynamodb.updateTable(params, (function(error, data) {
+                if (error || !data) {
+                    return this.emit('error', 'Fatal Error. Failed to update table. ' + error);
+                }
+            }).bind(this));
             setTimeout(dynamodb.describeTable.bind(dynamodb, { TableName: this.options.table }, this._checkTableReady.bind(this)), 1000);
```

Comment:
No I think its fine. The `_checkTableReady()` callback will keep looping until table is ready so all good there 👍